src/bodywork/cli/cli.py

```diff
@@ -358,16 +344,12 @@ def deployment(args: Namespace) -> None:
         print_warn("Please specify --name for the deployment job.")
         sys.exit(1)
     if command == "create":
+        load_kubernetes_config()
         if not async_workflow:
-            pass_through_args = Namespace(
-                git_url=git_url,
-                git_branch=git_branch,
-                bodywork_docker_image=image,
-            )
             print_info("Using local workflow controller - retries inactive.")
-            workflow(pass_through_args)
+            run_workflow(git_url, git_branch, docker_image_override=image)
```

Comment:
We need to run this in a `try ... except` block and look for `BodyworkWorkflowExecutionError` just like we did for the original `workflow` function.

Reply:
Should this not be done for all commands or was the intention to suppress any workflow errors being spat out again to the CLI?

Reply:
I can't remember

Reply:
No exceptions are meant to get printed to the CLI - we want everything to fail gracefully, where possible.

We also want to ensure that the command exists with non-zero exit code, as this may be used in CICD pipelines, etc.

Reply:
The non-zero aspect is fine but that means we should wrap all of the commands in a try/catch block.

Reply:
We do for k8s API exceptions, by virtue of the `handle_k8s_exceptions` decorator. All of the other CLI functions deliberately don't throw exceptions - they handle them internally.

Reply:
ahhhh I forgot about the decorator function we had on everything else.